src/index.js

```diff
@@ -31,9 +29,6 @@ export const HANDLERS = {
   404: noopHandler,
   '404-external': notFoundExternalDigestHandler,
   '404-internal': notFoundInternalDigestHandler,
-  'broken-backlinks': noopHandler,
```

Comment:
Here you can leave the noopHandler, because this will stay in post-processor until we completely deprecate it

Reply:
Added it back.